script.js

```diff
@@ -22,6 +102,12 @@ document.addEventListener('DOMContentLoaded', function() {
     return null;
   }
 
+  // Set up clear functionality for the search field
+  searchForm = document.querySelector("form[role='search']");
+  searchInput = searchForm.querySelector("input[type='search']");
```

Comment:
This will not work if there are multiple `{{search}}` helpers on the same page,  e.g. in the header and footer.
Could we adapt this code to run on every search form/input on the page?